README.md

```diff
@@ -10,9 +10,13 @@ This module uses the recommended way of passing sensitive data from SecretManage
 module "secrets" {
   source  = "exlabs/ecs-secrets-manager/aws"
   # We recommend pinning every module to a specific version
-  # version     = "x.x.x"
-  name = "data-pipeline-secrets"
-  ecs_task_execution_role = "ecs-task-execution-role"
+  version = "1.0.0"
```

Comment:
What is the reason behind uncommenting that line?

Before, it was required to check the version and install the proper one - which was not an ideal experience. Right now we are suggesting version `1.0.0` which doesn't exist

Reply:
@wilqq 
1. In my view, it would be better to suggest always the latest version.
2. After merging my changes I can create a new release for version `1.0.0`
3. Why version `1.0.0`? I think that it is good because we completely change a lot of the existing resources and their names, so some migration is required. Because of it, we can call that "breaking changes".

Reply:
Ok, we are using this module on production in a few projects, so it can be called `1.0.0` 🚀